lib/tabular_form_builder.rb

```diff
@@ -70,25 +60,118 @@ def #{selector}(field, options = {})
   end
 
   def select(field, choices, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
   end
 
   def collection_select(field, collection, value_method, text_method, options = {}, html_options = {})
-    label_for_field(field, options) + super
+    html_options[:class] = Array(html_options[:class]) + %w(form--select)
+
+    label_for_field(field, options) + container_wrap_field(super, 'select', options)
+  end
+
+  def collection_check_box(field,
+                           value,
+                           checked,
+                           text = field.to_s + "_#{value}",
+                           options = {})
+
+    label_for = "#{field}_#{value}".to_sym
+
+    input_options = options.reverse_merge(multiple: true,
+                                          checked: checked,
+                                          for: label_for,
+                                          label: text)
+
+    check_box(field, input_options, value, '')
+  end
+
+  def fields_for_custom_fields(record_name, record_object = nil, options = {}, &block)
+    options_with_defaults = options.merge(builder: CustomFieldFormBuilder)
+
+    fields_for(record_name, record_object, options_with_defaults, &block)
   end
 
   private
 
+  attr_reader :template
+
+  TEXT_LIKE_FIELDS = [
+    'number_field', 'password_field', 'url_field', 'telephone_field', 'email_field'
+  ].freeze
+
+  def container_wrap_field(field_html, selector, options = {})
+    ret = content_tag(:span, field_html, class: field_container_css_class(selector))
+
+    field_container_wrap_field(ret, options)
+  end
+
+  def field_container_wrap_field(field_html, options = {})
+    if options[:no_label]
+      field_html
+    else
+      content_tag(:span, field_html, class: 'form--field-container')
+    end
+  end
+
+  def localize_field(field, options, meth)
+    localized_field = Proc.new do |translation_form, multiple|
```

Comment:
Unused block argument - `multiple`. If it's necessary, use `_` or `_multiple` as an argument name to indicate that it won't be used.